api-reference/10 UI Widgets/dxFileManager/5 File System Providers/FileSystemProviderBase/3 Methods/abortFileUpload().md

```diff
@@ -15,7 +15,7 @@ id: FileSystemProviderBase.abortFileUpload()
 <!-- Description goes here -->
```

Comment:
There are empty descriptions in this file and those that follow it.

Reply:
they will be described in the next PR